### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,62 +8,36 @@ IdentityModel Extensions for .NET provide assemblies that are interesting for we
...
-* **IdentityModel Extensions for .NET 5.x**, **ASP.NET Core 1.x**, **ASP.NET Core 2.x**, **Katana 4.x**
-* **IdentityModel Extensions for .NET 4.x**, **ASP.NET 4**, **Katana 3.x**
-All other combinations aren't supported.
+We bumped the release from 6.x to 7.x.
+We are maintaining two releases from two different branches.
+dev - 7.x
+dev6x - 6.x
 
-For more details see [Migration notes here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/Migrating-from-Katana-(OWIN)-3.x-to-4.x)
+dev6x will be maintained until March 2024, at which point, you will need to move to 7x to continue to get the latest and greatest improvements and security updates.
```
Comment (jennyf19) [inline README.md:22]: would like feedback on this.

**Review by ciaozhang — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jmprieur) [reply README.md:22]: March = now + 6 months?
Maintain = ? only security fixes?

**Review by jmprieur — APPROVED** [review]
LGTM

not sure if we want to do dual write for all features? or only security fixes until March?

File: README.md (hunk 1 above)
Comment (jennyf19) [reply README.md:22]: Yes...not sure what was done in the past. I don't think we have any specific guidance on this.

**Comment by jennyf19** [comment]
> LGTM
> 
> not sure if we want to do dual write for all features? or only security fixes until March?

where do you see "dual"? @jmprieur

**Comment by jmprieur** [comment]
dual = 6x and 7x

File: README.md (hunk 1 above)
Comment (brentschmaltz) [reply README.md:22]: We originally committed to 1 year.

**Pull request merged**